drivers/wdm/audio/backpln/portcls/pin_wavecyclic.cpp

```diff
@@ -308,7 +298,7 @@ PinWaveCyclicAddLoopedStreamEvent(
     DPRINT1("Added event\n");
 
     // insert item
-    (void)ExInterlockedInsertTailList(&Pin->m_EventList, &EventEntry->ListEntry, &Pin->m_EventListLock);
+    (VOID)ExInterlockedInsertTailList(&Pin->m_EventList, &EventEntry->ListEntry, &Pin->m_EventListLock);
```

Comment:
... and that one could stay as they originally were.

Reply:
I moved the 4 remaining cases to a second commit: trivial, for consistency.

Reply:
Reverted the 4 remaining cases.